src/service/video-manager-impl.js

```diff
@@ -502,6 +505,10 @@ class VideoEntry {
       this.videoLoaded()
     );
     listen(video.element, VideoEvents.PAUSE, () => this.videoPaused_());
+    listen(video.element, VideoEvents.PLAY, () => {
```

Comment:
I asked this question offline. `play` event should be strictly before `playing` event, but can we guarantee this if the event are passed via postMessages.